src/screens/profile/index.tsx

```diff
@@ -110,6 +111,13 @@ const Profile = () => {
           isLoading={isLoading}
         />
       </ScrollView>
+      <ModalWithInput
+        onSubmit={() => {}} // TODO: Change password functionality
+        firstInputName={'Email'}
```

Comment:
this names what you pass for the inputs should be lower case

Reply:
done